haproxy_exporter.go

```diff
@@ -102,6 +102,8 @@ var (
 		16: newServerMetric("redispatch_warnings_total", "Total of redispatch warnings.", nil),
 		17: newServerMetric("up", "Current health status of the server (1 = UP, 0 = DOWN).", nil),
 		18: newServerMetric("weight", "Current weight of the server.", nil),
+		21: newServerMetric("check_fail_total", "Total number of failed health checks.", nil),
```

Comment:
We usually use the plural `check_fails_total` or `check_failures_total`, `failed_checks_total`. Pick one :)